gokart/build.py

```diff
@@ -94,6 +100,33 @@ def _reset_register(keep={'gokart', 'luigi'}):
     ]
 
 
+class TaskDumpMode(enum.Enum):
+    TREE = 'tree'
+    TABLE = 'table'
+    NONE = 'none'
+
+
+class TaskDumpOutputType(enum.Enum):
+    PRINT = 'print'
+    DUMP = 'dump'
+    NONE = 'none'
+
+
+@dataclass
+class TaskDumpConfig:
+    mode: TaskDumpMode = TaskDumpMode.NONE
+    output_type: TaskDumpOutputType = TaskDumpOutputType.NONE
+
+
+if sys.version_info < (3, 10):
```

Comment:
Added a quick fix to handle Python versions below 3.10. We'll streamline this once we drop Python 3.9 support.